rerun_py/rerun_sdk/rerun/_log.py

```diff
@@ -86,9 +95,13 @@ def log(
     entity:
         Anything that can be converted into a rerun Archetype.
     *extra:
-        An arbitrary number of additional component bundles.
+        An arbitrary number of additional component bundles, usually constructed with [`rerun.AnyValues`][].
```

Comment:
You could add two archetypes that play nice together like Points3D and Arrows3D. Usually seems a bit strong